docs/resources/advanced_cluster.md

```diff
@@ -679,8 +679,7 @@ In addition to all arguments above, the following attributes are exported:
     - DELETING
     - DELETED
     - REPAIRING
-* `replication_specs` - List of settings that configure your cluster regions. This array has one object per shard representing node configurations in each shard. For replica sets there is only one object representing node configurations. If for each replication_spec `num_shards` is configured > 1 (using legacy sharding schema), then each object represents a zone with one or more shards. To learn more see the [replication_specs argument reference](#replication_specs). This setting relies on the following computed attributes:
```

Comment:
removing from here so it is described in a single place (under arguments section)